sql/core/src/main/java/org/apache/spark/sql/execution/UnsafeFixedWidthAggregationMap.java

```diff
@@ -225,16 +220,20 @@ public void free() {
 
   @SuppressWarnings("UseOfSystemOutOrSystemErr")
   public void printPerfMetrics() {
```

Comment:
I can't find anyplace actually uses this method. Not sure if we want to remove it.